### DISCUSSION TITLE
各位，1.13版本可用，目前1.15不可用的可以退回

### DISCUSSION THREAD

**Discussion by zsanjin-p** [Harry-zklcdc/go-proxy-bingai#228] [Ideas]
亲测可用

**Comment by luckyEason** [comment]
> 亲测可用

1.15也能用，改一下核心api就行
https://newbing.fishhome.link
1.15不能用的原因是必应更新了api

**Comment by luckyEason** [comment]
![image](https://github.com/Harry-zklcdc/go-proxy-bingai/assets/109937362/3b3fa917-0814-4507-be8d-d3f327bc40b3)
